app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_ls = command.split()
+    if len(command_ls) == 3 and command_ls[0] == "cp" and command_ls[1] != command_ls[2]:
```

Comment:
Code Efficiency: You should add an additional check to verify that the source file exists before trying to open it. You can use the `os.path.isfile()` function to accomplish this. If the file does not exist, you should return an error message.